### DISCUSSION TITLE
Non-resizable window = Hide Maximize button?

### DISCUSSION THREAD

**Discussion by veniamin-ilmer** [rust-windowing/winit#2987] [General]
Looking in the [changelog](https://github.com/rust-windowing/winit/blob/master/CHANGELOG.md#version-0162-2018-07-07), in version 0.16.2, there is a comment:

> On Windows, non-resizable windows now have the maximization button disabled. This is consistent with behavior on macOS and popular X11 WMs.

Yet, if I try your [resize example](https://github.com/rust-windowing/winit/blob/master/examples/resizable.rs), being non-resizable, doesn't affect the maximize button.

So I want to confirm, is this a bug or a change done on purpose?

**Comment by kchibisov** [comment]
It depends on which platform you've tried it. It should be disabled iirc. It also depends what type of non-resizable window.

For example on X11 this wont work, because winit has no control over buttnos.

In general open an issue with more information about the system you're testing, etc.
